airbyte-webapp/src/pages/SourcesPage/pages/SourceItemPage/SourceItemPage.tsx

```diff
@@ -34,10 +24,6 @@ const SourceItemPage: React.FC = () => {
   );
 
   const source = useGetSource(params.id || "");
-  const sourceDefinition = useSourceDefinition(source.sourceDefinitionId);
-
-  // We load only connections attached to this source to be shown in the connections grid
-  const { connections } = useConnectionList({ sourceId: [source.sourceId] });
```

Comment:
All this was moved into the SourceOverviewPage. Should improve the speed of the settings page!